extension/src/popup/components/manageAssets/ManageAssetRowButton/index.tsx

```diff
@@ -95,7 +97,6 @@ export const ManageAssetRowButton = ({
   const walletType = useSelector(hardwareWalletTypeSelector);
   const networkDetails = useSelector(settingsNetworkDetailsSelector);
   const publicKey = useSelector(publicKeySelector);
-  const { recommendedFee } = useNetworkFees();
```

Comment:
small optimization here - because we render one of these components for every asset we show, we were hitting the fees endpoint many times in a short amount of time.

Changed to call the fee endpoint once in the top level and just pass it to all the child rows